html/changelogs/burgerbb-eventtweaks.yml

```diff
@@ -0,0 +1,7 @@
+author: BurgerBB
+
+delete-after: True
+
+changes: 
+  - rscadd: "Adds a new event, "Space-time Anomalies", which gives 1-3 lucky people either split personality, imaginary friend, or bluespace prophet."
```

Comment:
Not sure if we should mention the number of people or the mental conditions in the changelog.
I generally prefer to let them figure out the details on their own.

Reply:
Also, you need to escape or loose the " of "Space-time Anomailes"
Give that a try:
\"Space-time Anomailes\"